pkg/errors/errors.go

```diff
@@ -10,9 +10,9 @@ type LoadBalancerNotFoundError struct {
 
 func (e *LoadBalancerNotFoundError) Error() string { return e.e }
 
-func NewLoadBalancerNotFoundError(lb string) error {
+func NewLoadBalancerNotFoundError() error {
 	return &LoadBalancerNotFoundError{
-		e: fmt.Sprintf("Could not find LoadBalancer %s", lb),
+		e: "Could not find LoadBalancer",
```

Comment:
Don't we want to include the name of the LB we can't find?

Reply:
The GCP methods never learn the name of the load balancer.  They just take the IPs from the Service object passed in.

Reply:
@mbarnes I'd support that change. Is that something you'd like to make in this PR, or separately?

Reply:
Sure, I'll add it here.